.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  parser: 'babel-eslint',
```

Comment:
I believe we do since they are still experimental, and eslint is not supporting experimental features yet. See: eslint/eslint#4096, ember-cli/ember-cli-eslint#148